pydrumscore/export.py

```diff
@@ -528,6 +529,34 @@ def add_note(chord, notedef):
     with open(save_path, "wb") as f:
         f.write(xml_str)
 
+    zip_file= zipfile.ZipFile("pydrumscore/_exported/" + metadata.workTitle + ".mscz", "w")
```

Comment:
Doing this means that every time we run the export, we will export *both* in `.mscx` and `.mscz`. Instead I think we should have the choice by giving an argument.

By default, the "true" export format for MuseScore is the zipped one. So we should by default export in `.mscz`, but accept an argument that exports with the old method instead, unzipped. Could be a flag like `-u` for "uncompressed", which would be used for unit tests and during development :)